batcher/aligned-batcher/src/lib.rs

```diff
@@ -1188,17 +1210,33 @@ impl Batcher {
             // informative error.
 
             // Now we update the user states related to the batch (proof count in batch and min fee in batch)
-            batch_state_lock.update_user_proof_count(addr, *proof_count)?;
-            batch_state_lock.update_user_max_fee_limit(addr, *max_fee_limit)?;
-            batch_state_lock.update_user_total_fees_in_queue(addr, *total_fees_in_queue)?;
+            batch_state_lock
```

Comment:
Does this process leave a consistent state?

Reply:
https://github.com/yetanotherco/aligned_layer/issues/1520